install.sh

```
@@ -1118,6 +1130,9 @@ Requires=docker.service
 
 [Service]
 Restart=always
+# Create tmpfs mount for HLS segments (only if not already mounted)
+ExecStartPre=/bin/mkdir -p ${CONFIG_DIR}/hls
+ExecStartPre=/usr/bin/sh -c '[ -n "$(findmnt -n ${CONFIG_DIR}/hls)" ] || /bin/mount -t tmpfs -o size=50M,mode=0755,uid=${HOST_UID},gid=${HOST_GID},noexec tmpfs ${CONFIG_DIR}/hls'
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)